tests/spec/core/images-spec.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+describe("Core - Images", () => {
+  it("adds natural width and height to images if not set", async () => {
+    const ops = {
+      config: makeBasicConfig(),
+      body:
+      `<section>
+         <img id="image-with-id" src="data/images/image1.jpg">
+         <img src="data/images/image2.jpg">
+         <img id="image-with-dimensions" height=100 width=200 src="data/images/image1.jpg">
```

Comment:
For testing, I have used two images stored in `tests/data/images`. However, I don't think this is what you would prefer. So I'll wait for your review on this

Reply:
other option is to use a data URL generated from a small image (e.g., 3x5 pixels)... just enough to give you unique values for width and height.